exampleCourse/questions/downloadFile/question.html

```diff
@@ -0,0 +1,8 @@
+<pl_question_panel>
+    <ul>
+        <li>This text file comes from <code>clientFilesCourse</code>: <pl_file_download file_name="data.txt" type="static" scope="course" /></li>
```

Comment:
I'm wondering if it would be clearer to replace `scope="course"` with `directory="courseFilesQuestion"`? I know it looks worse, but it's more explicit. Otherwise we use two different ways of referring to the same thing. In the docs we will say "put files in the courseFilesQuestion directory", but then in the code we use the term "scope" and the keyword "question".